### ISSUE TITLE
`AVIF` format status

### ISSUE THREAD

**Issue by onkoe** [jay3332/ril#34]
Hey there! Just wanted to quickly check in on the status of `AVIF`. Would you be interested in any contributions to get it into the library? 

`image` supports some `AVIF` files, which makes it a better option for a general, inclusive use case, particularly when interacting with user images directly. However, this library benefits a lot from hindsight, and I adore the comfy API! 

`AVIF` has some issues with portability if you use the reference implementation, but `rav1e` might help there. (strong name!)

- Encoding: Taken care of with `rav1e` :)
- Decoding: [This is a bit harder](https://github.com/image-rs/image/pull/1398)! Seems to require `mp4parse` which can have difficulties with some niche files. I'm unaware of alternatives here. 

It's also worth considering that `image` [doesn't yet support 10-bit `AVIF`](https://github.com/image-rs/image/issues/2232).

**Comment by jay3332** [comment]
Yes, AVIF support would be greatly appreciated. It was originally planned to be after the initial stable release (1.0), but obviously there is no harm in getting it in RIL before so.

**Comment by onkoe** [comment]
thank you for the super helpful description! i'll take these into consideration :)
